packages/nightingale-interpro-track/src/nightingale-interpro-track.ts

```diff
@@ -258,6 +302,51 @@ class NightingaleInterproTrack extends NightingaleTrack {
     return expanded ? this.getFeatureColor(f) : "white";
   }
 
+  private getTextLabel(datum: Feature): string | null {
+    if (this.label?.length) {
+      if (this.label.startsWith(".")) {
+        return _get(datum, this.label.slice(1), null);
+      }
+      return this.label;
+    }
+    return datum?.feature?.accession || null;
+  }
+
+  private refreshLabels(base: LabelGroup) {
+    base
+      .attr("x", (f) => {
+        const start = f.start || 0;
+        const end = f.end || 0;
+        return this.getXFromSeqPosition(start + (end - start) / 2 || 1);
+      })
+      .attr(
+        "y",
+        (f) =>
+          this["margin-top"] +
+          (this.layoutObj?.getFeatureYPos(f.feature as Feature) || 0) +
+          (this.layoutObj?.getFeatureHeight(f.feature as Feature) || 0) / 2
+      )
+      .attr("fill", (_f, i, nodes) => {
+        const element = nodes[i];
+        const firstPath = element.parentElement?.querySelector("path.feature");
+
+        if (firstPath) {
+          return contrastingColor(getColor(firstPath, "fill"));
+        }
+        return null;
+      })
+      .attr(
+        "font-size",
+        (f) => (this.layoutObj?.getFeatureHeight(f.feature as Feature) || 2) - 2
+      )
```

Comment:
Makes the child labels _very_ small. Is the `-2` really necessary?

Reply:
I thought of it as a bit of padding, 1 pixel up and 1 down.
I'm making it that the children don't have padding